app/Models/Model.php

```diff
@@ -59,4 +59,12 @@ public function saveOrExplode($options = [])
 
         return $result;
     }
+
+    public function save(array $options = [])
+    {
+        $result = parent::save($options);
+        priv_clear_cache();
```

Comment:
I feel like this shouldn't be on the base model? Should just call it as needed? A lot of unnecessary clearing is gonna happen... (also why is this even needed?)

Reply:
Changes to model may change authorization.

Reply:
which models? shouldn't it only be done where it is necessary?

Reply:
there's currently only manual way to figure out whether it is necessary.

1. priv_check on a model
2. update the model
3. priv_check again for view

If `2` changes result of priv check, the cache must be cleared at one point before `3` is done. Placing it in controller can also work but it means either clearing the cache everywhere before rendering the view (incl. transformer) or checking whether priv check is used by the view, already cached, and affected by the just updated model.